Whiskey/Whiskey.psd1

```diff
@@ -170,6 +170,7 @@
 * Removed all support for old "VersionFrom", "PrereleaseMap", and "Version" properties in the root of your whiskey.yml file. Use Whiskey's `Version` task instead.
 
 ## Improvements and Bug Fixes
+

```

Comment:
Missing release notes

Reply:
This is a fix for new functionality that hasn't been released on master yet.

Reply:
👍